### PR TITLE
Command !push

### PR DESCRIPTION
**Pull request by nauar**
Solves issue #27

### CODE REVIEW

**Review by Seekfried — CHANGES_REQUESTED** [review]
You need to add this new command to cmdresults.json and README.md. But looks good otherwise.

File: cmdresults.json
Comment (Seekfried) [inline cmdresults.json:1]: In "helpadmin" missing

**Review by Seekfried — CHANGES_REQUESTED** [review]
See comment in cmdresults.json

File: cmdresults.json
Comment (nauar) [reply cmdresults.json:1]: Thanks! Added :)

**Review by Seekfried — APPROVED** [review]
Looks good

**Pull request merged**